text/0249-v2-experiments.md

```diff
@@ -374,6 +380,257 @@ Disadvantages:
    although something is to be said for it being the same with `private` fields
    today.
 
+#### POC results
+
+Option 5 was rejected since it relies on the use of prerelease qualifiers in a
+way that does not adhere to [Sematic Versioning](https://semver.org/). According
+to the semver [specification](https://semver.org/#spec-item-11), a precedence
+between versions is calculated **only** if the **major**, **minor** and
+**patch** are equal. This means that there is no way for a constructs library to
+declare a range of supported versions which include an experimental version.
+Sticking to the above example, a CDK construct library declaring a peer
+dependency on version `^1.60.0` of `aws-cdk-lib`, can not be used in a CDK
+application that declares a dependency on `aws-cdk-lib` version
+`1.61.0-experimental`. This is because the patch part in `1.60.0` and
+`1.61.0-experimental` is not equal, which means `1.61.0-experimental` does not
+satisfy `^1.60.0`. In npm versions prior to npm-v7, if `peerDependencies`
+requirements are not met, executing `npm install` would only issue a warning. In
+npm-v7, which automatically tries to install `peerDependencies`, executing
```

Comment:
Can we actually update [this section](https://github.com/aws/aws-cdk-rfcs/blob/master/text/0249-v2-experiments.md#prerelease-tags) with these findings about `npm-v7`?

Reply:
I can change the structure - sure. If it's just cosmetics, I prefer we do it after we agree on the best path forward. If it's something else, and you think this option needs more discussion,  I can edit it.

Reply:
I just want to make 100% sure we discard all 3 vending variants of option 5. (see https://github.com/aws/aws-cdk-rfcs/pull/279#discussion_r549891589).